gitops/sre/example/environment/compute/README.md

```diff
@@ -0,0 +1,17 @@
+# Cluster configuration
+
+This directory contains the `kustomization.yaml` that
+controls the various clusters' setup.
+
```

Comment:
Not only. It should also contain overlays: patches, additional manifests but there is none here.